### PR TITLE
[terra-alert] Fixed inconsistent height and custom color issue in fusion theme

### LINKED ISSUE #2788: [terra-alert][fusion theme] Custom Box Shadow Overrides Prop Defined Color
# Bug Report

## Description
In the fusion theme, the custom box-shadow color that is defined via the `customColorClass` prop is being overridden by the fusion class var `--terra-alert-custom-box-shadow` value. 

In the default style sheet, the color was not included in the custom variant's box shadow to allow the color class to be inherited: https://github.com/cerner/terra-core/blob/master/packages/terra-alert/src/Alert.module.scss#L127-L129

In the fusion theme, a color is defined for the box shadow, should it is not inheriting the custom color value.

## Steps to Reproduce

1. Go to https://engineering.cerner.com/terra-core/components/terra-alert/alert/alert
2. Look at the custom color class example
3. Change the theme to the `orion-fusion-theme`
4. See the custom color not being honored

## Additional Context / Screenshots
Screen Shot 2019-12-26 at 8 14 50 AM

## Expected Behavior
Screen Shot 2019-12-26 at 8 14 57 AM

## Solution
Update the [theme var](https://github.com/cerner/terra-core/blob/master/packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss#L38) to remove the color:

```diff
    --terra-alert-custom-background-color: #f4f4f4;
-  --terra-alert-custom-box-shadow: inset 2px 0 0 #9b9fa1;
+  --terra-alert-custom-box-shadow: inset 2px 0 0;
```

### LINKED ISSUE #2795: [terra-alert][fusion-theme] Alerts has Inconsistent Height
# Bug Report

## Description
Alerts in the fusion theme are , with the Dismiss button rendered from the `onDismiss` prop, or with terra-buttons supplied via `actions` prop render with a larger height than alerts with non-buttons.

### Height of Alert Without Button (expected)
Screen Shot 2020-01-08 at 3 32 19 PM

### Height of Alert With Button (wrong)
Screen Shot 2020-01-08 at 3 32 40 PM

## Expected Behavior
Alert heights should be consistent with or without buttons in them.

### PR DESCRIPTION
**Pull request by saket2403**
### Summary
Fixes #2788 and Fixes #2795

### CODE REVIEW

**Comment by ShettyAkarsh** [comment]
@saket2403 you have to update the same changes to all our theming repo's

File: packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss [hunk 1]
```diff
@@ -17,10 +17,10 @@
   --terra-alert-title-float: left;
   --terra-alert-title-margin-bottom: 0;
   --terra-alert-title-margin-right: 0.5rem;
-  --terra-alert-actions-margin-bottom: 1rem;
+  --terra-alert-actions-margin-bottom: 0.6158rem;
   --terra-alert-actions-margin-left: 0;
   --terra-alert-actions-margin-right: 0.5rem;
-  --terra-alert-actions-margin-top: 1rem;
+  --terra-alert-actions-margin-top: 0.6158rem;
```
Comment (ShettyAkarsh) [inline packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss]: What i am seeing is 0.658rem makes the alert height equivalent to default theme. can you test that one again ? also cross check once in clinical theme whether the same issue exists.

Comment (saket2403) [reply packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss]: In default theme the alert height with button is 42px and without button is 41px.
In fusion theme I have made both the alerts to be 41px. Should I make all variants as 41px or keep 41px without button and 42px with button.
cc @jeremyfuksa

Comment (jeremyfuksa) [reply packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss]: If everything can be 41px, then we should aim for consistency.

Comment (saket2403) [reply packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss]: In that case, I will update default theme with button to be 41px also

Comment (saket2403) [reply packages/terra-alert/themes/orion-fusion-theme/orion-fusion-theme.scss]: Updated in 378a40a

**Comment by emilyrohrbough** [comment]
Hey @saket2403  This issue wasn't ready to be picked up yet; I had logged it to make sure it didn't get forgotten. There were a few details that needed to be smoothed out and we were waiting on feedback from UX. We have decided we will be closing the recent alert issues I have logged in favor of picking up: https://github.com/cerner/terra-core/issues/1882. I will be taking on the work of evaluating terra-alert and bring it up to OCS standards and fixing the fusion theme issues I had logged. Thanks for picking this up and getting it started!!

//cc @ryanthemanuel @neilpfeiffer

**Pull request closed without merging**